Practicum/Pract. 03/zad1.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <fstream>
+
+constexpr char FILE_NAME[] = "FileWithLInes.txt";
+
+unsigned countLines(std::ifstream& in)
+{
+	char ch;
+	unsigned count = 0;
+	while (in.get(ch))
+	{
+		if (ch == '\n')
+			count++;
+	}
+	if (ch != '\n' && in.eof())
+	{
+		count++;
+	}
```

Comment:
you should always update the count

Reply:
check the implementation here 
https://github.com/GeorgiTerziev02/Object-oriented_programming_FMI/blob/main/Sem.%2002/Solutions/NewLinesCount.cpp